app/connectors/EmailConnector.scala

```diff
@@ -32,20 +33,20 @@ import scala.util.control.NonFatal
 class EmailConnector @Inject()(appConfig: FrontendAppConfig, httpClient: HttpClientV2)
                               (implicit ec: ExecutionContext) extends Logging {
 
-  def send(sendEmailRequest: SendEmailRequest)(implicit hc: HeaderCarrier): Future[Boolean] =
+  def send(sendEmailRequest: SendEmailRequest)(implicit hc: HeaderCarrier): Future[Done] =
     httpClient.post(url"${appConfig.emailServiceUrl}/hmrc/email")
       .withBody(Json.toJson(sendEmailRequest))
       .execute[HttpResponse]
       .flatMap { response =>
         response.status match {
-          case ACCEPTED => Future.successful(true)
+          case ACCEPTED => Future.successful(Done)
           case status =>
             logger.warn(s"Send email failed with status: $status")
-            Future.successful(false)
+            Future.successful(Done)
```

Comment:
Test for when any status other than 202 is returned.

Reply:
There is a test for this already as discussed.